ddtrace/span.py

```diff
@@ -359,6 +365,15 @@ def set_tags(self, tags):
             for k, v in iter(tags.items()):
                 self.set_tag(k, v)
 
+    @property  # type: ignore
+    @deprecated(version="1.0.0")
```

Comment:
can we used `ddtrace.vendor.debtcollector.removals.removed_property` instead?

```suggestion
    @removed_property(removal_version="1.0.0", ....)
```

Reply:
Using `removed_property` was my initial approach but it raised a mypy error. This is my work around. I'll give it another shot

Reply:
what was the error? it _should_ work?

Reply:
This code block:

```
 def meta(self):
        return self._meta

    @meta.setter
    def meta(self, value):
        self._meta = value
```

raised this typing error: ` Name "meta" already defined on line 368  [no-redef]`

Using `# type: ignore` on @meta.setter worked for me

Reply:
hmm, weird that mypy doesn't pick up on it, maybe because it is a `@removed_property` and not `@property`.

Instead of just a blanket `ignore`, can you ignore only `[no-redef]`?

Reply:
I updated that line to # type: ignore[no-redef]